components/EventCard/EventCard.component.js

```diff
@@ -31,7 +31,7 @@ const EventCard = (props) => {
                     <Image style={styles.imagePopUp} resizeMode="contain" source={props.source} />
                     <Text style={styles.datePopUp}> {props.date} </Text>
                     <Text> {props.link} </Text>
-                    <TouchableOpacity style={styles.btnPopUp}>
+                    <TouchableOpacity style={styles.btnPopUpRSVP}>
                         <Text style={styles.btnText} 
                         onPress={
                             () => {
```

Comment:
Use the custom Button called "Main Button" in our components.

Reply:
Oh okay, my bad. Did not see that. I will make that adjustment. Thanks!

Reply:
Wait, the "MainButton" adjusts every other button. I was focusing more on the buttons when the card pop-ups. Unless I'm mistaking something.

Reply:
You could add custom styling to the MainButton component. It's fine, I can fix it. lol